video_getter_cv2.py

```diff
@@ -224,6 +225,18 @@ def reconnect(self):
         self.pauseTime = None
         self.start()
 
-    def get_frame_time(self):
-        #Returns time elapsed since start of video, in miliseconds
-        return self.stream.get(cv2.CAP_PROP_POS_MSEC)  
\ No newline at end of file
+    def get_frame_time(self, clock):
+        """
+        Parameters
+        ----------
+        clock : utils.clock.Clock object
+
+        Returns
+        ----------
+         - If source_type is a file, returns time elapsed since start of video in miliseconds
+         - Else, returns current unix time, in milliseconds
+         """        
+         if self.source_type == 'file':
+            return self.stream.get(cv2.CAP_PROP_POS_MSEC)
+        else: 
+            return int(1000 * clock.get_now_SGT_unixts())
```

Comment:
wow didn't know abt `clock.get_now_SGT_unixts()`

Reply:
ok learnt that it was custom code by @levan92 

updated @jxk20 that this module should be able to work as a standalone module